examples/tutorials/05_ordinary_torch_models.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<p>Do not stop too abruptly. Add some text at the end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>